### ISSUE TITLE
Apply the trained model to unseen data.

### ISSUE THREAD

**Issue by yh-zhu** [anbai106/mlni#7]
I want to use the trained model(final model) to predict unseen data.
Is there a way to do it?

**Comment by anbai106** [comment]
The software saved the trained model in a certain format. What you need to do is to read the saved model and apply the model to unseen data. We did not include this as a functionality because this is quite straightforward and depends on the user's choice.

**Comment by yh-zhu** [comment]
Thank you. I have tried to set the save_model option as True/False, but I have trouble to figure out which one is the final model. 

When the 'save_model' option was set to False, it resulted in saving the last repetition, is this the final model(ex. svm-1_last_repetition.joblib )? 
When it was set to True, there were 100 models for each CV, was the cv_99 the final model?

I also tried to load the model to get a prediction for unseen data, however it did not give me the subtypes label, but only predicted the new data as PT(1).
